src/index.js

```diff
@@ -288,7 +288,9 @@ export default class ImageTool {
     this._data.caption = data.caption || '';
     this.ui.fillCaption(this._data.caption);
 
-    Tunes.tunes.forEach(({ name: tune }) => {
+    const tunes = Tunes.tunes.concat(this.config.actions);
```

Comment:
Hi @yiyenene
This config.actions already transfer in tunes by [here](https://github.com/editor-js/image/blob/master/src/index.js#L143-147)
And if it is a bug can you please describe what kind of problem may occur.

Reply:
@robonetphy `Tunes.tunes` only returns default tunes (withBorder, stretched and withBackground) because it is a static method.
it might be a better way to add a new method that returns merged tunes (default tunes and custom actions) to `Tune`🤔 

> can you please describe what kind of problem may occur.

1. Add custom action (name, icon, title)
2. Run editor.js and add new image block
3. Open the menu on the right side, and you will see the button of custom action has `active` style nevertheless data is undefined.